### PR TITLE
ABC-96 Add new functionality for custom emojis

### PR DESCRIPTION
**Pull request by jwilander**
#### Summary
* Add sorting for get custom emojis actions
* Add actions for searching and autocompleting custom emojis

DO NOT MERGE UNTIL AFTER mattermost/mattermost-server#8125

#### Ticket Link
Part of https://mattermost.atlassian.net/browse/ABC-96

#### Checklist
- [x] Added or updated unit tests (required for all new features)
- [x] All new/modified APIs include changes to the drivers

### CODE REVIEW

File: src/actions/emojis.js [hunk 1]
```diff
@@ -53,7 +54,7 @@ export function getAllCustomEmojis(perPage = General.PAGE_SIZE_MAXIMUM) {
                     emojis = await Client.getCustomEmojis();
                     hasMore = false;
                 } else {
-                    emojis = await Client4.getCustomEmojis(page, perPage);
+                    emojis = await Client4.getCustomEmojis(page, perPage, 'name');
```
Comment (enahum) [inline src/actions/emojis.js]: what does 'name' mean?

Comment (jwilander) [reply src/actions/emojis.js]: Sorting by name, I can make it a constant if that would make it clearer

Comment (enahum) [inline src/actions/emojis.js]: what does 'name' mean? are there any other possible options?

Comment (jwilander) [reply src/actions/emojis.js]: Sorting by name, and right now it's the only option other than having no sorting

Comment (enahum) [reply src/actions/emojis.js]: Well lets make that a constant if possible

**Comment by jwilander** [comment]
@enahum done

**Pull request merged**